src/foam/dao/EasyDAO.js

```diff
@@ -197,9 +207,10 @@ return delegate;
     },
     {
       /** Keep a history of all state changes to the DAO. */
-      class: 'Boolean',
-      name: 'journaled',
-      value: false
+      class: 'foam.core.Enum',
+      of: 'foam.dao.JournalType',
+      name: 'journalType',
+      value: 'JournalType.NO_JOURNAL'
```

Comment:
I don't think this works but you might be getting NO_JOURNAL by luck because it's the first value for the enum. I think you want.
```
value: 'NO_JOURNAL'
```

Reply:
Also flag this for java only
```
flags: ['java']
```